### DISCUSSION TITLE
Suggestions

### DISCUSSION THREAD

**Discussion by qawechi** [masterking32/MasterCryptoFarmBot#22] [General]
![IMG_20240929_101621.jpg](https://github.com/user-attachments/assets/a38025e7-c985-4713-ae05-e6bea7276249)

- The bot logs' output font is so big, we always have to scroll right/left and up/down to see everything.

- If the bot logs' output would be in real time, it is so much better! So we don't have to click on refresh logs every time.

![IMG_20240929_095527.jpg](https://github.com/user-attachments/assets/aec0933c-8066-4230-8ad7-bb369844ddeb)

- The licence here is not wrapped, if it was a disabled (read-only) text input, is so much better.

![IMG_20240929_095940.jpg](https://github.com/user-attachments/assets/2a3f9191-cdf6-413e-b06b-6f3ea992d33c)

- these configuration attributes would be easier and more user friendly if u guys put them into the gui. For example, a section named "Configuration", instead of being in the config.py file.

- Add more payment methods, plz!

- Add language support :) so the user can change the language of the ui.

![IMG_20240929_101639.jpg](https://github.com/user-attachments/assets/8a6e979c-7cee-460c-b9d0-eee32d1be95c)

- these two buttons are big, i mean, the width of the columns.

- the terms of service are not included in the gui.

**Comment by Fy0urM** [comment]
Hide your ip in, in browser header

**Reply by qawechi** [reply]
Fixed that. Thanks!

**Comment by Fy0urM** [comment]
Not a good idea to throw the panel port out into the public
